install/devsds/lib/opensds.sh

```diff
@@ -84,7 +84,7 @@ osds::opensds::install(){
     export OPENSDS_AUTH_STRATEGY=$OPENSDS_AUTH_STRATEGY
     export OPENSDS_ENDPOINT=http://localhost:50040
     build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "block"}'
-    build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "file", "provisioningProperties":{"ioConnectivity": {"accessProtocol": "NFS"},"DataStorage":{"StorageAccessCapability":["Read","Write","Execute"]}}}'
+    build/out/bin/osdsctl profile create '{"name": "default", "description": "default policy", "storageType": "file", "provisioningProperties":{"ioConnectivity": {"accessProtocol": "nfs"},"DataStorage":{"StorageAccessCapability":["Read","Write","Execute"]}}}'
```

Comment:
Can u confirm if it's `NFS` or `nfs`?

Reply:
It is not required now, source code will take to create accessprotocol default as 'nfs', if you don't provide these values. Better remove accessprotocol and storageaccesscapability. It is done in PR#883

Reply:
Ok, I will delete it.